src/cplscheme/BaseCouplingScheme.hpp

```diff
@@ -373,6 +373,11 @@ class BaseCouplingScheme : public CouplingScheme {
    */
   void determineInitialReceive(DataMap &receiveData);
 
+  /**
+   * @brief returns true, if current iteration is in the last time window.
+   */
+  bool isLastTimeWindow() const;
```

Comment:
The name of this function is confusing. I would expect this function to be true when we reached the last time window of the simulation.